src/message-backup/BackupClient.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Where message backups should be stored
+ */
+export enum BackupType {
+  none,
+  xmtp,
+}
+export interface BackupProvider {
+  type: BackupType
+}
+export type SelectBackupProvider = () => Promise<BackupProvider>
+
+export interface BackupConfiguration {
```

Comment:
What is the relation ship between BackupConfiguration and Account. Is it a strict 1:1 relationship?